test-projects/configs/flat-js/eslint.config.mjs

```diff
@@ -0,0 +1,58 @@
+import globals from 'globals';
+import js from '@eslint/js';
+import e from 'eslint-plugin-ember';
+
+import ember from 'eslint-plugin-ember/configs/recommended';
+import emberGJS from 'eslint-plugin-ember/configs/recommended-gjs';
+
+import emberParser from 'ember-eslint-parser';
+import babelParser from '@babel/eslint-parser';
+
+export default [
+  js.configs.recommended,
+  {
+    linterOptions: {
+      reportUnusedDisableDirectives: 'error',
+    },
+  },
+  {
+    files: ['**/*.js'],
+    languageOptions: {
+      parser: babelParser,
+      parserOptions: {
+        ecmaFeatures: { modules: true },
+        ecmaVersion: 'latest',
+        requireConfigFile: false,
+        babelOptions: {
+          plugins: [['@babel/plugin-proposal-decorators', { decoratorsBeforeExport: true }]],
+        },
+      },
+      globals: {
+        ...globals.browser,
+      },
+    },
+    plugins: {
+      ember: e,
+    },
+    rules: {
+      ...ember.rules,
+      ...emberGJS.rules,
+    },
+  },
+  {
+    files: ['**/*.gjs'],
+    languageOptions: {
```

Comment:
Gjs needs the same parser config like *.js

Reply:
It's normally not a problem since a config file is used

Reply:
O guess we should document it. That internally babel parser is used. Or should we just add some default plugins?

Reply:
I think we should document it, and then let eslint-plugin-ember decide if some utilities or defaults would be helpful to configure